backend/model/person/router.js

```diff
@@ -0,0 +1,14 @@
+const controller = require('./controller');
+const Router = require('express').Router;
+const router = new Router();
+
+router.route('/')
+  .get((...args) => controller.find(...args))
```

Comment:
Justing wondering if this could just be:
```
router.route('/')
  .get(controller.find)
  .post(controller.create)
```

Reply:
probably. again, didn't think about how it was coded too much, but if i were writing it from scratch, i would've likely done it your way